src/main/java/frc/robot/Constants.java

```diff
@@ -27,10 +27,24 @@ public final class Constants {
 
   // Maximum speed of the robot in meters per second, used to limit acceleration.
 
-  public static final class AutoConstants {
-
+  public static final class PivotConstants {
     public static final PIDConstants TRANSLATION_PID = new PIDConstants(0.7, 0, 0);
     public static final PIDConstants ANGLE_PID = new PIDConstants(0.4, 0, 0.01);
+    public static final double PIVOT_kP = 0;
+    public static final double PIVOT_kI = 0;
+    public static final double PIVOT_kD = 0;
+    public static final double PIVOT_kS = 0;
+    public static final double PIVOT_kG = 0;
+    public static final double PIVOT_kV = 0;
+    public static final double PIVOT_kA = 0;
+    public static final int PIVOT_MOTOR = 20;
+    public static final double PIVOT_MAX_VEL = 4;
+    public static final double PIVOT_MAX_ACCEL = 20;
+    public static final double PIVOT_WEIGHT = 9.55;
+    public static final double MOTOR_ZERO_SPEED = 0;
+    public static final int MOTOR_CURRENT_LIMIT = 40;
+    public static final double PIVOT_MAX = -(Math.PI / 2) - 0.02;
+    public static final double PIVOT_MIN = 0.08;
```

Comment:
I don't think we use these

Reply:
Removed PIVOT_MIN; PIVOT_MAX is used in Command PivotUp